includes/class-newspack-newsletters.php

```diff
@@ -1208,7 +1208,7 @@ public static function display_newsletters_in_archives( $query ) {
 		if ( is_admin() || ! $query->is_main_query() ) {
 			return;
 		}
-		if ( $query->is_archive() && ! $query->is_post_type_archive() ) {
+		if ( ( $query->is_tag() || $query->is_category() ) && $query->is_archive() ) {
```

Comment:
is_tag and is_category are always archives... isn't the last check redundant?

Reply:
You're right! Fixed in f4871e6792380c89b2a1f30bdb0ee71f01a1d3ed